### PR TITLE
Clarify prerequisites Composer and yarn for ddev

### PR DESCRIPTION
**Pull request by sypets**

### CODE REVIEW

File: Documentation/Appendix/SettingUpTypo3Ddev.rst [hunk 1]
```diff
@@ -41,7 +41,10 @@ Then install DDEV as described in the
 `Installation <https://ddev.readthedocs.io/en/latest/#installation>`__
 instructions.
 
-In order to be able to build some assets (e.g. css, js files) you need to have `yarn` installed as `described here <https://yarnpkg.com/lang/en/docs/install/>`__.
+Composer can run inside of ddev, so there is no need to set it up locally.
+
+In order to be able to build some assets (e.g. css, js files) you still need to have `yarn`
```
Comment (susannemoog) [inline Documentation/Appendix/SettingUpTypo3Ddev.rst:46]: hmm, `yarn` is also available inside ddev and should run via `ddev exec yarn` without the need to install it on the host.

Comment (sypets) [reply Documentation/Appendix/SettingUpTypo3Ddev.rst:46]: The last time I tested this (quite a while ago), yarn was not yet available inside DDEV.

> In order to be able to build some assets (e.g. css, js files) you need to have yarn installed as described here.

can be removed

This needs some more changes:

* [ ] should also be mentioned on https://docs.typo3.org/m/typo3/guide-contributionworkflow/master/en-us/Setup/Prerequisites.html and https://docs.typo3.org/m/typo3/guide-contributionworkflow/master/en-us/Setup/SetupTypo3.html (readers are already referred to DDEV for Composer, should be mentioned that this goes for yarn too. In fact they can skip over those pages)
* [ ] link to the DDEV page from there

Comment (susannemoog) [reply Documentation/Appendix/SettingUpTypo3Ddev.rst:46]: yes, could be that this is a newer addition. I tested earlier today and it worked fine. 
to your first point: I'd suggest adding a tip box on top, that the whole section can be skipped when using ddev and link to the ddev page from that box.

Comment (sypets) [reply Documentation/Appendix/SettingUpTypo3Ddev.rst:46]: I agree. Already on it ...

**Pull request merged**